hoot-rnd/src/test/cpp/hoot/rnd/conflate/frechet/FrechetSublineMatcherTest.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+
+// CPP Unit
+#include <cppunit/extensions/HelperMacros.h>
+#include <cppunit/extensions/TestFactoryRegistry.h>
+#include <cppunit/TestAssert.h>
+#include <cppunit/TestFixture.h>
+
+// GEOS
+#include <geos/geom/LineString.h>
+// Hoot
+#include <hoot/rnd/conflate/frechet/FrechetSublineMatcher.h>
+#include <hoot/core/io/OsmReader.h>
+#include <hoot/core/io/OsmWriter.h>
+#include <hoot/core/visitors/FindWaysVisitor.h>
+#include <hoot/core/MapProjector.h>
+
+#include <hoot-core-test/src/test/cpp/hoot/core/TestUtils.h>
+
+using namespace hoot;
+
+
+namespace hoot
+{
+
+class FrechetSublineMatcherTest : public CppUnit::TestFixture
+{
+  CPPUNIT_TEST_SUITE(FrechetSublineMatcherTest);
+//  CPPUNIT_TEST(sublineTest);
```

Comment:
They are for now.  I've got to come up with some good tests which will follow soon.  You want me to do them first before committing the code?